### DISCUSSION TITLE
For reproducibility,how can I  fix the output?

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#168] [Q&A]
**For reproducibility,how can I  fix the output?**
It is said in the help of function rfsrc:
![image](https://user-images.githubusercontent.com/5112397/149262193-d7db2795-99c2-4c73-977a-b64214717917.png)
**but the below code does not work--It changes each runs:**
library(randomForestSRC)
print(rfsrc(Species~.,iris,importance=TRUE, seed=-1)$importance)

**Comment by kogalur** [comment]
This is a duplicate.  Please see Issue #67.
